<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>984</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>550</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>7</number>
      </property>
      <widget class="QWidget" name="tabWidgetPage1">
       <attribute name="title">
        <string>Ajouter</string>
       </attribute>
       <widget class="QPushButton" name="btnAjouter">
        <property name="geometry">
         <rect>
          <x>740</x>
          <y>400</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="CIN_input">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>40</y>
          <width>221</width>
          <height>31</height>
         </rect>
        </property>
        <property name="inputMask">
         <string>99999999</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="NOM_input">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>120</y>
          <width>221</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="PRENOM_input">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>210</y>
          <width>221</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="MAIL_input">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>300</y>
          <width>221</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QDateTimeEdit" name="dateTimeEdit">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>60</y>
          <width>221</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="doubleSpinBox">
        <property name="geometry">
         <rect>
          <x>780</x>
          <y>120</y>
          <width>62</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>10</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>CIN</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>90</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>NOM</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>170</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>PRENOM</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>260</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>MAIL</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>30</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Date Reservation</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>120</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Duree en heure</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>260</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Employer</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>180</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Voiture</string>
        </property>
       </widget>
       <widget class="QComboBox" name="select_voiture">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>211</y>
          <width>231</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="select_Employer">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>300</y>
          <width>231</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tabWidgetPage2">
       <attribute name="title">
        <string>List reservation</string>
       </attribute>
       <widget class="QTableView" name="afficher_reservation">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>921</width>
          <height>311</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>700</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1000</width>
          <height>1000</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_CIN_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>51</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
        <property name="maxLength">
         <number>8</number>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>20</y>
          <width>161</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Recherche avec CIN</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radio_CIN">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>60</y>
          <width>93</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>CIN</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radio_alpha">
        <property name="geometry">
         <rect>
          <x>660</x>
          <y>60</y>
          <width>93</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>ALPHABET</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radio_Date">
        <property name="geometry">
         <rect>
          <x>830</x>
          <y>60</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>DATE HEURE</string>
        </property>
       </widget>
       <widget class="Line" name="line">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>90</y>
          <width>951</width>
          <height>16</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="Telecharger_button">
        <property name="geometry">
         <rect>
          <x>860</x>
          <y>10</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Telecharger</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="progressBar">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>440</y>
          <width>271</width>
          <height>24</height>
         </rect>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tabWidgetPage3">
       <attribute name="title">
        <string>Supprimer</string>
       </attribute>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>60</y>
          <width>181</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ID_Reservation</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnSupprimer">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>180</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="supp_ID">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>100</y>
          <width>211</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Modifier</string>
       </attribute>
       <widget class="QComboBox" name="Employer_select">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>310</y>
          <width>251</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>300</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>MAIL</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>90</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>CIN</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_Nom">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>180</y>
          <width>231</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>60</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Date Reservation</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>160</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>NOM</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
        <property name="geometry">
         <rect>
          <x>730</x>
          <y>170</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>280</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Employer</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>200</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Voiture</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_CIN">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>110</y>
          <width>231</width>
          <height>41</height>
         </rect>
        </property>
        <property name="inputMask">
         <string>99999999</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_prenom">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>250</y>
          <width>231</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>230</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>PRENOM</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>170</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Duree en heure</string>
        </property>
       </widget>
       <widget class="QComboBox" name="Voiture_select">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>230</y>
          <width>251</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_Mail">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>330</y>
          <width>231</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QDateTimeEdit" name="dateTimeEdit_2">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>110</y>
          <width>251</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>20</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_ID">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>40</y>
          <width>231</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="btnModifier">
        <property name="geometry">
         <rect>
          <x>750</x>
          <y>400</y>
          <width>181</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Nom des clients</string>
       </attribute>
       <widget class="QLabel" name="label_20">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>231</width>
          <height>51</height>
         </rect>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="palette">
         <palette>
          <active>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>85</red>
              <green>85</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>85</red>
              <green>85</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>85</red>
              <green>85</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>NOMBRE DES CLIENTS:</string>
        </property>
       </widget>
       <widget class="QLCDNumber" name="lcdNumber">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>110</y>
          <width>461</width>
          <height>181</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="nb_client">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>120</y>
          <width>251</width>
          <height>181</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Afficher</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Statistique</string>
       </attribute>
       <widget class="QFrame" name="chart_graph">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>30</y>
          <width>821</width>
          <height>461</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="mailing">
       <attribute name="title">
        <string>Envoyer un mail</string>
       </attribute>
       <widget class="QLineEdit" name="mail_titre">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>170</y>
          <width>211</width>
          <height>51</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="combo_reservation">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>70</y>
          <width>211</width>
          <height>61</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_21">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>40</y>
          <width>171</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Select une Reservation</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_22">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>140</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Titre</string>
        </property>
       </widget>
       <widget class="QPushButton" name="envoyer_mail">
        <property name="geometry">
         <rect>
          <x>740</x>
          <y>390</y>
          <width>201</width>
          <height>71</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Envoyer</string>
        </property>
       </widget>
       <widget class="QCalendarWidget" name="calender_reservation">
        <property name="geometry">
         <rect>
          <x>505</x>
          <y>40</y>
          <width>411</width>
          <height>231</height>
         </rect>
        </property>
       </widget>
       <widget class="QProgressBar" name="progressBar_3">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>320</y>
          <width>211</width>
          <height>31</height>
         </rect>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Historique</string>
       </attribute>
       <widget class="QTableView" name="afficher_historique">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>881</width>
          <height>371</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QProgressBar" name="progressBar_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>440</y>
          <width>231</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
       <widget class="QPushButton" name="Telecharger_historique">
        <property name="geometry">
         <rect>
          <x>800</x>
          <y>10</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Telecharger</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Coordinates</string>
       </attribute>
       <widget class="QTableView" name="coordinates">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>100</y>
          <width>881</width>
          <height>251</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QLabel" name="label_23">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>53</y>
          <width>311</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Mongolian Baiti</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Coordinates GPS avec Arduino</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Changer_coor">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>410</y>
          <width>151</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>Changer coordonnes</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Envoyer_coor">
        <property name="geometry">
         <rect>
          <x>732</x>
          <y>410</y>
          <width>151</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>Envoyer Arduino</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>984</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuReservation">
    <property name="title">
     <string>Reservation</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuanother_one">
    <property name="title">
     <string>another one</string>
    </property>
   </widget>
   <addaction name="menuReservation"/>
   <addaction name="menuanother_one"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
